common/authen.js

```diff
@@ -174,6 +179,7 @@
                     document.body.style.cursor = 'default';
                     $window.location = '../login?referrer=' + UriUtils.fixedEncodeURIComponent(referrer);
                 });
+                // TODO: Conclude chain w/ a .catch for catchAll
```

Comment:
Instead of redirecting user to log in again. Reject error cb with catchAll here.